recipes/unicore/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -x -e
+
+SOURCE_BIN="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}/bin/unicore"
+TARGET_BIN="${PREFIX}/bin/unicore"
+
+pip install torch transformers sentencepiece protobuf
```

Comment:
Installing dependencies with `pip` is not good!
Please use:
- https://anaconda.org/conda-forge/pytorch
- https://anaconda.org/conda-forge/transformers
- https://anaconda.org/conda-forge/sentencepiece
- https://anaconda.org/conda-forge/protobuf
as `host` (and `run` ?!) dependencies

Reply:
Makes sense, thanks for the recommendation :) We will check our code's integrity with these packages and push another commit.